### ISSUE TITLE
WebExtension does not pass the submission because of an unexpected error

### ISSUE THREAD

**Issue by cosminbadescuSV** [mozilla/addons-linter#693]
This is a follow up to https://github.com/mozilla/addons-server/issues/2699

Steps to reproduce:

1.In AMO home page hover over “Tools”
2.Click on “Submit a New Add-on”
3.After submission page is loaded click on “Select a file...”
4.Upload [Kifi.zip](https://github.com/mozilla/addons-linter/files/277101/Kifi.zip) file .

Expected results:
The WebExtension is successfully submitted.

Actual results:
The WebExtension cannot be submitted because of an unexpected error.

Note/Issues:
Verified on FF49.0a1 (2016-05-22) (Win 7 64-bit). This issue is reproducing in AMO-dev and AMO-stage.
Screenshots to this issue:
![unex1](https://cloud.githubusercontent.com/assets/15885467/15465046/43232f54-20db-11e6-8ac7-dcba95b810dc.gif)
![unex2](https://cloud.githubusercontent.com/assets/15885467/15465050/465ba138-20db-11e6-8f0b-e43154d1ba80.gif)

**Comment by EnTeQuAk** [comment]
This is the stack trace I get validating the add-on locally:
```
[git:master (olympia)] ~/Projects/olympia> addons-linter kifi/Kifi.xpi --log-level info --stack
{"name":"AddonLinterJS","hostname":"dubby","pid":2869,"level":30,"msg":"Creating new linter instance { config: \n   { _: [ 'kifi/Kifi.xpi' ],\n     metadata: false,\n     pretty: false,\n     stack: true,\n     boring: false,\n     'self-hosted': false,\n     selfHosted: false,\n     help: false,\n     h: false,\n     'log-level': 'info',\n     logLevel: 'info',\n     output: 'json',\n     o: 'json',\n     max: '35',\n     libraries: 'node_modules/addons-linter/src/libraries.json',\n     '$0': 'node_modules/.bin/addons-linter' } }","time":"2016-05-24T11:45:17.600Z","v":0}
{"name":"AddonLinterJS","hostname":"dubby","pid":2869,"level":30,"msg":"Package is a file. Attempting to parse as an .xpi/.zip","time":"2016-05-24T11:45:17.607Z","v":0}
{"name":"AddonLinterJS","hostname":"dubby","pid":2869,"level":30,"msg":"Retrieving metadata from manifest.json","time":"2016-05-24T11:45:17.641Z","v":0}
{"name":"AddonLinterJS","hostname":"dubby","pid":2869,"level":30,"msg":"Metadata option is set to false","time":"2016-05-24T11:45:17.898Z","v":0}
Error: CSS url() "initial" is invalid
    at extractCSSUri (/home/chris/Projects/olympia/node_modules/addons-linter/dist/webpack:/src/utils.js:223:11)
    at isLocalCSSUri (/home/chris/Projects/olympia/node_modules/addons-linter/dist/webpack:/src/utils.js:230:21)
    at Object.detectBadMozBindingURL (/home/chris/Projects/olympia/node_modules/addons-linter/dist/webpack:/src/rules/css/detectBadMozBinding.js:11:13)
    at CSSScanner.processCode (/home/chris/Projects/olympia/node_modules/addons-linter/dist/webpack:/src/scanners/css.js:50:7)
    at /home/chris/Projects/olympia/node_modules/addons-linter/dist/webpack:/src/scanners/css.js:64:22
    at run (/home/chris/Projects/olympia/node_modules/core-js/modules/es6.promise.js:89:22)
    at /home/chris/Projects/olympia/node_modules/core-js/modules/es6.promise.js:102:28
    at flush (/home/chris/Projects/olympia/node_modules/core-js/modules/_microtask.js:18:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by tofumatt** [comment]
Oddly I just tried the attachment locally and didn't get an error 🤔

**Comment by tofumatt** [comment]
Nevermind: I didn't have the addons-linter waffle switch on. Now I see it 👍

**Comment by tofumatt** [comment]
Looks like the code that's failing is worried about [-moz-binding](https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-binding), which is XUL/XBL anyway. Web Extensions shouldn't have access to them I'm assuming, so actually this rule should be deleted and then this should be fixed.

I'm on it!

**Comment by cosminbadescuSV** [comment]
This isssue is not reproducing anymore, here is a video:
![kifi](https://cloud.githubusercontent.com/assets/15885467/15601101/92f9a2d8-23f6-11e6-9789-b031092cfc39.gif)
